discord/webhook/sync.py

```diff
@@ -705,7 +705,7 @@ def from_url(cls, url: str, *, session: Session = MISSING, bot_token: Optional[s
             A partial webhook is just a webhook object with an ID and a token.
         """
         m = re.search(
-            r"discord(?:app)?.com/api/webhooks/(?P<id>[0-9]{17,20})/(?P<token>[A-Za-z0-9\.\-\_]{60,68})",
+            r"discord(?:app)?.com/api/webhooks/(?P<id>\d{17,20})/(?P<token>[A-z\d.\-_]{60,68})",
```

Comment:
```suggestion
            r"discord(?:app)?.com/api/webhooks/(?P<id>\d{17,20})/(?P<token>[\w\.\-_]{60,68})",
```
These could be shortened